test/functionalTest/cases/3122_location_metadata_issues/create_location_attr_V1.test

```diff
@@ -0,0 +1,215 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Create 2 entities with NGSIV1 POST. One with the location metadata (defining a location attribute
+according to the "old way"), the other without a location metadata (and then without location attribute)
+Then check both entities in DB.
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. Create E1 with NGSIV1
+# 02. Create E2 with NGSIV1
+# 03. Check E1 entity in DB
+# 04. Check E2 entity in DB
+
```

Comment:
I would suggest to use more precise steps (including echo messages) in this test. For instance

```
# 01. Create E1 with NGSIV1 with location metadata
# 02. Create E2 with NGSIV1 without location metadata
# 03. Check E1 entity in DB has location metadata and location GeoJSON field
# 04. Check E2 entity in DB has no location metadata nor location GeoJSON field
```

Reply:
Fixed in c76622d